frontend/pages/queries/ManageQueriesPage/ManageQueriesPage.tsx

```diff
@@ -112,34 +113,55 @@ const ManageQueriesPage = ({
   );
   const [showInheritedQueries, setShowInheritedQueries] = useState(false);
 
+  interface IQueryKeyQueriesLoadAll {
+    scope: "enhancedQueries";
+    teamId: number | undefined;
+  }
+
   const {
     data: curTeamEnhancedQueries,
     error: curTeamQueriesError,
     isFetching: isFetchingCurTeamQueries,
     refetch: refetchCurTeamQueries,
-  } = useQuery<IListQueriesResponse, Error, IEnhancedQuery[]>(
-    [{ scope: "queries", teamId: teamIdForApi }],
-    () => queriesAPI.loadAll(teamIdForApi),
+  } = useQuery<
+    IListQueriesResponse,
```

Comment:
Should the type for this be `IEnhandedQuery[]`? I believe it refers to what is returned by the query function, which in this case would mean the result of the chained operation.

Reply:
Good call